### ISSUE TITLE
My work on tuning hyperparameters 

### ISSUE THREAD

**Issue by liuchuan22** [EGalahad/legged_gym#1]
## work have done
Start position is changed to `[0.0, 0.0, 0.28]`. This makes the dog's foot touch the ground initially.
Default joint angles is adjusted to 

    default_joint_angles = {  # = target angles [rad] when action = 0.0
            "FL_hip_joint": 0.0,  # [rad]
            "RL_hip_joint": 0.0,  # [rad]
            "FR_hip_joint": 0.0,  # [rad]
            "RR_hip_joint": 0.0,  # [rad]
            "FL_thigh_joint": -40 / 57.3,  # [rad]
            "RL_thigh_joint": -48 / 57.3,  # [rad]
            "FR_thigh_joint": -40 / 57.3,  # [rad]
            "RR_thigh_joint": -48 / 57.3,  # [rad]
            "FL_calf_joint": 45 / 57.3,  # [rad]
            "RL_calf_joint": 30 / 57.3,  # [rad]
            "FR_calf_joint": 45 / 57.3,  # [rad]
            "RR_calf_joint": 30 / 57.3,  # [rad]
        }
This is (I guess) most likely the real state of the dog.
Now my `cyberdog2.py` is modified to train walking on a flat ground. You may have to change `play_vis.py` to modify the visualization of the dog.
Friction is changed to default value (0.2 is significantly too small).
Whether using `curriculum` learning or not does not affect the final result much.
The `self.collisions` is disabled now. I think this improves the dog's performance a little bit.
## some existing problems
1. The `base_height_target` is set to `0.26` and I observed that the loss is small ($\sqrt{0.002}$ m, about 5 cm), but in visualization the dog is never walking on that height. Maybe this should be double checked.

2. Why our dog is moving so slow? In the video of other robots like A1 and Anymal, they move very fast, but our dog is somehow struggling.

3. Why our dog moves on its elbows instead of its feet? I guess maybe this is the urdf's problem.

https://github.com/EGalahad/legged_gym/assets/113412566/309c3e24-2c3e-4838-9fb4-5ed503340bcc
